go.mod

```diff
@@ -11,12 +11,14 @@ require (
 	github.com/coreos/go-systemd/v22 v22.3.3-0.20220203105225-a9a7ef127534
 	github.com/docker/go-units v0.4.0
 	github.com/dolmen-go/contextio v0.0.0-20200217195037-68fc5150bcd5
-	github.com/elastic/e2e-testing v1.99.2-0.20220117192005-d3365c99b9c4
+	github.com/elastic/beats/v7 v7.17.7
```

Comment:
Ugh, I didn't notice this come in. We have been trying to keep Beats out of the agent dependency list. I'll see if I can get rid of this.

Reply:
Had to clear out my local build/package directory.